requirements.txt

```diff
@@ -65,7 +65,7 @@ unicodecsv==0.14.1
 virtualenv==15.0.3
 virtualenv-clone==0.2.6
 virtualenvwrapper==4.7.2
-wagtail==1.7
+wagtail>=1.8,<1.9
```

Comment:
What's the reason we don't just use `==1.8` - is it because we want the latest minor version?

Reply:
We want the latest patch version. The minor version is `1.8.x`, which won't change. But we want patch/bug fixes that are applied to the minor version we are on.

Reply:
see: http://docs.wagtail.io/en/v1.13.1/releases/upgrading.html#upgrade-process

Reply:
Cool, that's what I assumed, thanks!